packages/peregrine/lib/talons/MiniCart/ProductList/productListFragments.gql.js

```diff
@@ -6,21 +6,24 @@ export const ProductListFragment = gql`
         items {
             id
             uid
+            # eslint-disable-next-line @graphql-eslint/require-id-when-available
```

Comment:
Oof, we're going to have these all over the place.

But I don't think we want to disable this rule completely either 😬

Reply:
@supernova-at Yep... I know.
graph-ql-eslint does have an [issue open](https://github.com/dotansimha/graphql-eslint/issues/382) to support multiple ID fields. This would solve the issue